index.bs

```diff
@@ -430,7 +430,8 @@ CommandData = (
   NetworkCommand //
   ScriptCommand //
   SessionCommand //
-  StorageCommand
+  StorageCommand //
+  WebExtensionsCommand
```

Comment:
Oh, we also should have used `WebExtensionCommand` here and `WebExtensionResult` below given that this is the name of the module.

Reply:
Opened - https://github.com/w3c/webdriver-bidi/pull/825